### ISSUE TITLE
[FVT]remove the dependency of perl-xCAT package by xCAT-test

### ISSUE THREAD

**Issue by tingtli** [xcat2/xcat-core#1871]
From xCAT 2.12.2 , we found xCAT-test requires perl-xCAT, since we would like to have xCAT-test as an independent package, we would like to remove the requires from xCAT-test and perl-xCAT

```
[root@c910f02c01p22 xcat-core]# rpm -qpR xCAT-test-2.12.3-snap201609200644.noarch.rpm
/bin/sh
/bin/sh
/bin/sh
/usr/bin/env
perl(Data::Dumper)
perl(File::Basename)
perl(File::Copy)
perl(File::Path)
perl(Getopt::Long)
perl(POSIX)
perl(Sys::Hostname)
perl(Term::ANSIColor)
perl(Time::Local)
perl(lib)
perl(strict)
perl(warnings)
perl(xCAT::Utils)
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(VersionedDependencies) <= 3.0.3-1
rpmlib(PayloadIsXz) <= 5.2-1
```

This is introduced by genesistest.pl.

```
[root@c910f02c01p18 xCAT-test]# grep Utils ./* -R
./autotest/testcase/genesis/genesistest.pl:use xCAT::Utils;
./autotest/testcase/genesis/genesistest.pl:my $os = xCAT::Utils->osver("all");
./autotest/testcase/genesis/genesistest.pl:my $master=xCAT::TableUtils->get_site_Master();
```

@junxiawang , please think about this and provide your fix for this. Thanks!

**Comment by whowutwut** [comment]
@junxiawang @tingtli  I don't think you can remove the requires from the rpm, because I believe `rpmbuild` auto detects these requirement when it's building the rpm and looking at the `use` statements in the code.   Which scenario do you need `xCAT-test` to be an independent package?

**Comment by tingtli** [comment]
@whowutwut  We could like to install xCAT-test in the test environment. Then, we could make xCAT installation as an independent case by using xcattest -t install_xCAT. Then, in the test log, we can see that if xCAT installation is succeeded or failed. 
If in Junxia's code, she does not use "use" statements. I think she can make it possible.

**Comment by tingtli** [comment]
Since this bug is for some enhancement for xCAT-test code. It is not required by 2.13. Move to next release.

**Comment by tingtli** [comment]
@junxiawang Do you have updates for this bug?

**Comment by junxiawang** [comment]
merged in #3266

**Comment by tingtli** [comment]
Verified this with latest build.
```
[root@]# lsxcatd -v
Version 2.13.5 (git commit 5f3e1659de2850f46b9135d49ea1f762eeeb7dd6, built Tue Jun 27 07:15:48 EDT 2017)

[root@]# rpm -qpR xCAT-test-2.13.5-snap201706270715.noarch.rpm
/bin/bash
/bin/sh
/bin/sh
/bin/sh
/usr/bin/env
perl(Data::Dumper)
perl(File::Basename)
perl(File::Copy)
perl(File::Path)
perl(Getopt::Long)
perl(POSIX)
perl(Sys::Hostname)
perl(Term::ANSIColor)
perl(Time::Local)
perl(lib)
perl(strict)
perl(warnings)
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
```
